src/Hazelcast.Net/HazelcastClient.cs

```diff
@@ -135,14 +136,17 @@ private void WireComponents()
         public HazelcastOptions Options => _optionsClone ??= _options.Clone();
         
         /// <inheritdoc />
-        public IReadOnlyCollection<MemberInfo> Members => Cluster.Members.GetMembers().ToList();
+        public IReadOnlyCollection<MemberInfoState> Members => Cluster.Members.GetMembersAndState().ToList();
```

Comment:
Can we leave old property for compatibility and add a new one, like `MembersState`?